docs/configuration/alerting_rules.md

```diff
@@ -32,16 +32,13 @@ groups:
 ```
 
 The optional `for` clause causes Prometheus to wait for a certain duration
-between first encountering a new expression output vector element and counting an alert as firing for this element. This helps ensure an alert is valid. In this case, Prometheus will ensure the expression output vector elements are consistent for 10 minutes before firing the alert. Elements that are active, but not firing yet, are in the pending state.
+between first encountering a new expression output vector element and counting an alert as firing for this element. In this case, Prometheus will check the expression output vector elements have been consistent each evaluation for 10 minutes before firing the alert. Elements that are active, but not firing yet, are in the pending state.
```

Comment:
I'd talk about this in the context of a single alert continuing to fire from cycle to cycle, rather than talking about the whole vector being "consistent". Each alert is processed independently.

Reply:
Okay - updated the text.